test/domain-shared/env.test.js

```diff
@@ -0,0 +1,5 @@
+describe("NODE_ENV", () => {
+    it("environment should be test", () => {
+        expect(process.env.NODE_ENV).toBe("test");
```

Comment:
Just nice to know that we can always depend on NODE_ENV to be development, production or test.
These can be used to differentiate logging filters, or level of detail for exceptions caught in an exception handler.